Yoast/wordpress-seo
# 10631 :: Extend Yoast to support oEmbed

This update extends the Yoast functionality by hooking into the oEmbed data and adding OG title & image information (if they are set).

## Summary

This PR can be summarized in the following changelog entry:

* Add support for oEmbed utilization of Yoast custom fields (post meta) values. Specifically the image and the title

## Test instructions
This PR can be tested by following these steps:

* Add a "facebook" image to a post that is different from the post's feature image and a title that is different then the post's title. 
* Browse to `/wp-json/oembed/1.0/embed?url=[YOUR_POSTS_ENCODED_URL_HERE]`.
* You should _now_ see the custom string and image (with dimensions) from the OG custom fields (post meta)

## Quality assurance

* [ x ] I have tested this code to the best of my abilities
* [ x ] I have added unittests to verify the code works as intended

Fixes #10631
